include/veriblock/blockchain/pop/pop_state_machine.hpp

```diff
@@ -170,10 +163,18 @@ struct PopStateMachine {
                  "its payloads");
     }
 
-    index.setFlag(BLOCK_APPLIED);
-    if (shouldSetCanBeApplied) {
+    // if the applied block count equals the size of the chain between the root
```

Comment:
```suggestion
    // if the applied block count equals the size of the chain between the bootstrap block
```

Reply:
there can be several bootstrap blocks, while the root block refers to the root of the tree

Reply:
ok, add a comment `the root block (first bootstrap block)`